src/cogs/split.py

```diff
@@ -0,0 +1,205 @@
+import json
+import random
+from dataclasses import dataclass, asdict
+from typing import Any, Optional
+
+from discord import VoiceChannel, Embed, Forbidden, Role, AllowedMentions
```

Comment:
This change is required if you accept the suggestion in https://github.com/fisher60/spliteam/pull/4#discussion_r699778564
```suggestion
from discord import VoiceChannel, Embed, Forbidden, Role, AllowedMentions, HTTPException
```